docs/data/introduction/licensing/licensing.md

```diff
@@ -0,0 +1,177 @@
+You can learn more about our stewardship ethos in [this document from our company handbook](https://mui-org.notion.site/Stewardship-542a2226043d4f4a96dfb429d16cf5bd).
+
+We offer commercial licenses to developers who need the most advanced features that cannot be easily maintained by the open-source community.
+Commercial licenses enable us to support a full-time staff of engineers, which is simply not possible through the MIT model.
+
+Rest assured that when we release features commercially, it's only because we believe that you will not find a better MIT-licensed alternative anywhere else.
+
+See the [Pricing](https://mui.com/pricing/) page for a detailed feature comparison.
+
+## Plans
+
+### Community Plan
+
+The free version of MUI X is [published under an MIT license](https://tldrlegal.com/license/mit-license) and is [free forever](https://mui-org.notion.site/Stewardship-542a2226043d4f4a96dfb429d16cf5bd#20f609acab4441cf9346614119fbbac1).
+This version contains features that we believe are maintainable by contributions from the open-source community.
+
+MIT licensed packages:
+
+- [`@mui/x-data-grid`](https://www.npmjs.com/package/@mui/x-data-grid)
+- [`@mui/x-date-pickers`](https://www.npmjs.com/package/@mui/x-date-pickers)
+
+### Pro Plan<span class="plan-pro"></span>
+
+The Pro version of MUI X expands on the features of the free version with more advanced capabilities such as multi-filtering, multi-sorting, column resizing and column pinning for the data grid; as well as the date range picker component.
+
+The Pro version is available under a commercial license—visit [the Pricing page](https://mui.com/pricing/) for details.
+
+Pro packages:
+
+- [`@mui/x-data-grid-pro`](https://www.npmjs.com/package/@mui/x-data-grid-pro)
+- [`@mui/x-date-pickers-pro`](https://www.npmjs.com/package/@mui/x-date-pickers-pro)
+
+:::info
+The features exclusive to the Pro version are marked with the <span class="plan-pro"></span> icon throughout the documentation.
+:::
+
+<div class="only-light-mode">
+  <img src="/static/x/commercial-header-icon-light.png" style="width: 579px; margin-bottom: 2rem;" alt="">
+</div>
+<div class="only-dark-mode">
+  <img src="/static/x/commercial-header-icon-dark.png" style="width: 560px; margin-bottom: 2rem;" alt="">
+</div>
+
+### Premium Plan<span class="plan-premium"></span>
+
+The Premium version of MUI X covers the most advanced features of the data grid, such as row grouping, Excel export, aggregation, in addition to everything that's included in the Pro Plan.
+
+The Premium version is available under a commercial license—visit [the Pricing page](https://mui.com/pricing/) for details.
+
+Premium package:
+
+- [`@mui/x-data-grid-premium`](https://www.npmjs.com/package/@mui/x-data-grid-premium)
+
+:::info
+The features exclusive to the Premium version are marked with the <span class="plan-premium"></span> icon throughout the documentation.
```

Comment:
Should we add an example image like we do for pro ?

Reply:
Actually I'm thinking we should remove the screenshot from the Pro section—I don't think it really adds any value.

Reply:
Could also make sense, or just inline an example if it's worth it